### PR TITLE
tools/rbd_nbd: add --version show support

### PR DESCRIPTION
**Pull request by TsaiJin**
There exists `--version`  option in the rbd-nbd help info.  
However, it doesn't work when run the command ｀rbd-nbd --version`

Signed-off-by: Jin Cai <caijin.caij@alibaba-inc.com>

### CODE REVIEW

**Comment by TsaiJin** [comment]
retest this please.

File: src/tools/rbd_nbd/rbd-nbd.cc [hunk 1]
```diff
@@ -932,6 +933,9 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg, Config *
   for (i = args.begin(); i != args.end(); ) {
     if (ceph_argparse_flag(args, i, "-h", "--help", (char*)NULL)) {
       return -ENODATA;
+    } else if (ceph_argparse_flag(args, i, "-v", "--version", (char*)NULL)) {
+      std::cout << pretty_version_to_str() << std::endl;
+       _exit(0);
```
Comment (joscollin) [inline src/tools/rbd_nbd/rbd-nbd.cc]: I don't understand why an error is returned for --help. As per the convention followed here, the option --version also should return an error. But Why can't they return a positive int values to distinguish --help and --version ? Then process inside rbd_nbd() and then return 0 from there ?

Comment (TsaiJin) [inline src/tools/rbd_nbd/rbd-nbd.cc]: @joscollin I change this pull request according to your advice.

**Comment by TsaiJin** [comment]
retest this please.

**Comment by joscollin** [comment]
@TsaiJin main returns `1` for `--help` though, but that is not related with this PR.

**Comment by joscollin** [comment]
Test Result:
```
[jcollin@earth build]$ ./bin/rbd-nbd -v
ceph version 12.1.0-761-gb7dec695ec (b7dec695ec84718e18f515be5435398fe9085729) luminous (rc)
[jcollin@earth build]$ ./bin/rbd-nbd --version
ceph version 12.1.0-761-gb7dec695ec (b7dec695ec84718e18f515be5435398fe9085729) luminous (rc)
```

**Comment by TsaiJin** [comment]
@joscollin The return value of `--help` is because the 'generic_server_usage()' function always exits with status 1

**Comment by joscollin** [comment]
@TsaiJin Yes, I have verified that. It is not related with this PR, so merging this.

**Comment by joscollin** [comment]
- http://pulpito.front.sepia.ceph.com/jcollin-2017-07-29_16:49:18-rbd-wip-jcollin-testing_29-07-2017-distro-basic-smithi/
- http://pulpito.front.sepia.ceph.com/compare/?branch=wip-jcollin-testing_29-07-2017&suite=rbd

**Pull request merged**